clusters/build-clusters/01_cluster/machine_config/m5d4x_machineconfig.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker-m5d4x
+  name: m5d4x
```

Comment:
worker-m5d4x

Reply:
That's fine, though it's a harmless change - nothing much cares about the *names* of MCs other than ordering.